Kitodo/src/main/java/org/kitodo/production/services/data/TaskService.java

```diff
@@ -294,7 +294,7 @@ public void replaceProcessingUser(Task task, User user) {
      * @return a list of titles
      */
     public List<String> findTaskTitlesDistinct() throws DataException, DAOException {
-        return findDistinctValues(QueryBuilders.matchAllQuery(), "title.keyword", true, countDatabaseRows());
+        return findDistinctValues(QueryBuilders.matchAllQuery(), "title.keyword", true, countDatabaseRows()); // TODO nur template tasks?
```

Comment:
If this comment still valid? If yes why is this not be decided while developing this?